src/main/java/ch/astorm/jotlmsg/io/FlatEntryStructureFactory.java

```diff
@@ -0,0 +1,23 @@
+package ch.astorm.jotlmsg.io;
+
+import java.lang.reflect.InvocationTargetException;
+import java.nio.ByteBuffer;
+
+/**
+ * Creates FlatEntryStructures.
+ * 
+ * @author Guido Stein
+ *
+ */
+public class FlatEntryStructureFactory<T extends FlatEntryStructure> {
+    public T createFlatEntryStructure(Class<T> type, ByteBuffer bf) {
+        T fes = null;
+        try {
+            fes = type.getDeclaredConstructor(new Class[] { ByteBuffer.class }).newInstance(bf);
+        } catch (InstantiationException | IllegalAccessException | IllegalArgumentException | InvocationTargetException
+                | NoSuchMethodException | SecurityException e) {
+            e.printStackTrace();
```

Comment:
Also here, letting the stacktrace just be printed would lead to and unseen problem if there is an exception... Add a `throw new RuntimeException(e)` instead.

Reply:
You are absolutely right. Bad idea! I will replace it. 👍